### PR TITLE
Fetch token image from CoinGecko

### PR DESCRIPTION
**Pull request by vbaranov**
Resolves https://github.com/blockscout/blockscout/issues/9025

## Motivation

Cache native coin image together with price.

## Changelog

Save native coin image from CoinGecko API to ETS.

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [x] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [x] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: apps/explorer/lib/explorer/market/market_history_cache.ex [hunk 1]
```diff
@@ -11,12 +11,22 @@ defmodule Explorer.Market.MarketHistoryCache do
 
   @cache_name :market_history
   @last_update_key :last_update
+  @image_last_update_key :image_last_update
   @history_key :history
+  @image_key :image
   # 6 hours
   @recent_days 30
 
+  def fetch_image(image_url) do
```
Comment (nikitosing) [inline apps/explorer/lib/explorer/market/market_history_cache.ex:20]: What is the purpose of this function? As I see image url already stored in cache within `Explorer.ExchangeRates.Token`

File: apps/explorer/lib/explorer/exchange_rates/token.ex [hunk 2]
```diff
@@ -17,6 +17,7 @@ defmodule Explorer.ExchangeRates.Token do
    * `:symbol` - Trading symbol used to represent a currency
    * `:usd_value` - The USD value of the currency
    * `:volume_24h_usd` - The volume from the last 24 hours in USD
+   * `:image` - Token image
```
[the remark this answers is not in the document]
Comment (vbaranov) [reply apps/explorer/lib/explorer/exchange_rates/token.ex]: Done in https://github.com/blockscout/blockscout/pull/9132/commits/62242f4103c45b06a3da4b1b09ac85272a6242a2

File: apps/explorer/lib/explorer/market/market_history_cache.ex (hunk 1 above)
Comment (vbaranov) [reply apps/explorer/lib/explorer/market/market_history_cache.ex]: You're right. It is redundant. Removed in https://github.com/blockscout/blockscout/pull/9132/commits/9d5b2afbe74027ffd0bcb9b9fa3a30ca236de99d.

**Review by nikitosing — APPROVED** [review]

**Review by sl1depengwyn — APPROVED** [review]

**Comment by BitrockChain** [comment]
Hi @vbaranov , is any environment change to display icon from coingecko? is it need EXCHANGE_RATES_COINGECKO_API_KEY?

**Pull request merged**